src/app/core/learning-object-module/hierarchy/hierarchy.service.ts

```diff
@@ -60,12 +59,11 @@ export class HierarchyService {
   /**
    * Adds children to a learning object
    *
-   * @param username the username of the author
    * @param object the object having children
    * @param children the children to be had
    * @returns
    */
-  async addChildren(username: string, object: any, children): Promise<any> {
```

Comment:
Is username being taken care of somewhere else or do we not need username at all?

Reply:
I don't think we ned username, was it not being referenced anywhere?

Reply:
Username parameter wasn't being referenced anywhere